projects/schematics/src/migrations/mechanism/update-ssr/update-ssr-files.ts

```diff
@@ -4,8 +4,42 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { chain, Rule } from '@angular-devkit/schematics';
+import {
+  chain,
+  Rule,
+  SchematicContext,
+  Tree,
+} from '@angular-devkit/schematics';
+import { Path } from '@angular-devkit/core';
+import {
+  SERVER_BAK_FILENAME,
+  SERVER_FILENAME,
+} from '../../../shared/constants';
 
 export function updateServerFiles(): Rule {
-  return chain([]);
+  return chain([removeServer]);
+}
+
+function removeServer(): Rule {
+  return (tree: Tree, _context: SchematicContext) => {
+    let serverPath;
+    let serverBakPath;
```

Comment:
let's add typing as in the case of assigning the value inside the `visit` loop the type won't be inferred:
![image](https://github.com/SAP/spartacus/assets/28891782/f82c8ba6-35c6-4310-ba97-8f4004407f7e)

```suggestion
    let serverPath: Path | undefined;
    let serverBakPath: Path | undefined;
```

Reply:
I've added types